packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -307,6 +328,19 @@ export class UnreferencedStateTracker {
     }
 }
 
+/**
+ * Error class raised when a SweepReady object is used, indicating a bug in how
+ * references are managed in the container by the application, or a bug in how
+ * GC tracks those references.
+ *
+ * There's a chance for false positives when this error is raised by a Main Container,
+ * since only the Summarizer has the latest truth about unreferenced node tracking
+ */
+class SweepReadyUsageError extends LoggingError implements IFluidErrorBase {
+    /** This errorType will be in temporary use (until Sweep is fully implemented) so don't add to any errorType type */
+    public errorType: string = "objectUsedAfterMarkedForDeletionError";
```

Comment:
Consider calling this something like `objectUsedAfterGarbageCollectedError` or something that suggests this object is gone. My concern is that if this conveys that object is only marked but not gone, it diminishes the impact and makes it sound like its our bug. What do you think?

Reply:
Yeah, I see that.  How about `unreferencedObjectUsedAfterGarbageCollected`?  I'm dropping the term "Error" at the end because it's so long and don't want it to sound like "after an error with GC"

Reply:
sounds good.